HW/spark.py

```diff
@@ -0,0 +1,104 @@
+import sys
+
+from pyspark.sql import Row,SparkSession
+from pyspark.sql.types import (
+    BooleanType,
+    DoubleType,
+    IntegerType,
+    StringType,
+    StructField,
+    StructType,
+)
+
+def main():
+    appName = "Baseball"
+    master = "local"
+    # Setup Spark
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+
+    # Create a column that has all the "words" we want to encode for modeling
+    # Each word must be in an array (hence SPLIT)
+    baseball_df = spark.sql(
+        """
+            create or replace temporary table joined_batter_game as
```

Comment:
Julien mentioned he wanted all logic to run in the spark code and to not modify the SQL database itself. I'm not sure if the create or replace line works in this context or where the temporary table would be saved to. But I think you can just select this using a WITH ... AS (SELECT ...) statement